easybuild/easyconfigs/n/novoalign/novoalign-3.09.00.eb

```diff
@@ -0,0 +1,52 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```

Comment:
@jdemeul Should be `easybuilders` rather than `hpcugent` now, see also failing test